Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -413,6 +413,15 @@ open class BasicFormat: SyntaxRewriter {
       anchorPoints[token] = currentIndentationLevel
     }
 
+    // Add an indentation in the line for the initial token,
+    // if the token is present and its leadingTrivia does not contain the indentation.
```

Comment:
I’m not sure if my formulation of the comment is more helpful than yours but I think some information like the following would have helped me.

````swift
// If we are formatting a subtree and the line that the initial token occurs on is indented, use that line indentation for the first token in the subtree to format. For example, when formatting only the code block in the following, then the opening `{` should be indented by four spaces.
// ```
//     func test() {
//         print(1)
//     }
// ```
````

Reply:
Nice comment! Thank you https://github.com/apple/swift-syntax/pull/1867/commits/a27a8d3ff0da5f40ca6c104377d3472b6cc0f728